grpc/diatheke.proto

```diff
@@ -76,6 +83,11 @@ service Diatheke {
   // valid intent or not.
   rpc PushText(PushTextRequest) returns (Empty) {}
 
+  // Set the current story for a running session. This function can
+  // be used to implement system intiated alerts or to change the current
+  // session state.
+  rpc SetStory(StoryRequest) returns (Empty) {}
```

Comment:
We are returning empty here, but should we tell them that responses should be coming on the other calls?

Reply:
When I look at documentation, I don't usually read _everything_.  I typically just go to the one function I care about and read that one.  So, having a small pointer to the event stream would be enough for me to remember to go read that.